hack/build/config.sh

```diff
@@ -26,8 +26,8 @@ FUNC_TEST_REGISTRY_POPULATE="cdi-func-test-registry-populate"
 FUNC_TEST_REGISTRY_INIT="cdi-func-test-registry-init"
 FUNC_TEST_BAD_WEBSERVER="cdi-func-test-bad-webserver"
 # update this whenever builder Dockerfile is updated
-BUILDER_TAG=${BUILDER_TAG:-0.0.5}
-BUILDER_IMAGE=${BUILDER_IMAGE:-kubevirt/kubevirt-cdi-bazel-builder@sha256:bed4f8c2451747f7f26eedce2b435b7538b9a0350b1dc4e928b005acc4c977a6}
+BUILDER_TAG=${BUILDER_TAG:-0.0.6}
+BUILDER_IMAGE=${BUILDER_IMAGE:-mhenriks/kubevirt-cdi-bazel-builder:0.0.6}
```

Comment:
Need to do the builder dance here, I also have a builder PR up, maybe I can combine what you need there?